### PR TITLE
Fixed bugs in history navigation in rollups and transform jobs pages and some UI changes

### PR DESCRIPTION
**Pull request by tandonks**
### Description

- Removed plus icons from create buttons for pages other than main pages
- Fixed the padding in the action dropdown buttons
- Fixed bug in history navigation in Transform and Rollup Details pages
- Fixed padding between bottom buttons

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/pages/RollupDetails/containers/RollupDetails/__snapshots__/RollupDetails.test.tsx.snap [hunk 1]
```diff
@@ -1,7 +1,583 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`<RollupDetails /> spec renders the component 1`] = `
-<div>
-  Testing rollup landing page
+<div
+  style="padding: 5px 50px;"
+>
+  <div
```
Comment (skumawat2025) [inline public/pages/RollupDetails/containers/RollupDetails/__snapshots__/RollupDetails.test.tsx.snap:7]: Why this file has so many changes?

File: public/pages/RollupDetails/containers/RollupDetails/RollupDetails.tsx [hunk 2]
```diff
@@ -125,12 +125,10 @@ export class RollupDetails extends Component<RollupDetailsProps, RollupDetailsSt
         ? [BREADCRUMBS.ROLLUPS, { text: id }]
         : [BREADCRUMBS.INDEX_MANAGEMENT, BREADCRUMBS.ROLLUPS, { text: id }];
       this.context.chrome.setBreadcrumbs(newBreadCrumbs);
-      this.props.history.push(`${ROUTES.ROLLUP_DETAILS}?id=${id}`);
```
Comment (skumawat2025) [inline public/pages/RollupDetails/containers/RollupDetails/RollupDetails.tsx:128]: Let's please make sure we are not regressing on anything. Please make sure if old UI is working as expected.

Comment (tandonks) [reply public/pages/RollupDetails/containers/RollupDetails/RollupDetails.tsx:128]: Checked

File: public/pages/RollupDetails/containers/RollupDetails/__snapshots__/RollupDetails.test.tsx.snap (hunk 1 above)
Comment (tandonks) [reply public/pages/RollupDetails/containers/RollupDetails/__snapshots__/RollupDetails.test.tsx.snap]: Done

**Comment by skumawat2025** [comment]
LGTM.

File: public/pages/CreateSnapshotPolicy/components/CronSchedule/CronSchedule.tsx [hunk 3]
```diff
@@ -120,6 +120,7 @@ const CronSchedule = ({
   let startTimeContent;
   startTimeContent = (
     <EuiDatePicker
+      compressed
```
Comment (skumawat2025) [inline public/pages/CreateSnapshotPolicy/components/CronSchedule/CronSchedule.tsx:123]: Thanks for adding this @tandonks.

**Review by SuZhou-Joe — APPROVED** [review]

**Pull request merged**